projects/generate_glitches/generate_glitches.py

```diff
@@ -103,38 +106,35 @@ def generate_glitch_dataset(
     - start: start gpstime
     - stop: stop gpstime
     - window: half window around trigger time to query data for
-    - sample_rate: sampling frequency
+    - sample_rate: sampling arequency
     - channel: channel name used to read data
     - frame_type: frame type for data discovery w/ gwdatafind
     - trig_file: txt file output from omicron triggers
             (first column is gps times, 3rd column is snrs)
     - vetoes: SegmentList object of times to ignore
     """
+
     glitches = []
     snrs = []
 
-    # snr and time columns in omicron file
-    snr_col = 2
-    time_col = 0
-
     # load in triggers
-    triggers = np.loadtxt(trig_file)
+    triggers = h5py.File(trig_file)["triggers"][()]
 
     # restrict triggers to within gps start and stop times
     # and apply snr threshold
-    times = triggers[:, time_col]
+    times = triggers["time"][()]
     mask = (times > start) & (times < stop)
-    mask &= triggers[:, snr_col] > snr_thresh
+    mask &= triggers["snr"][()] > snr_thresh
```

Comment:
Does it make sense to do these in an `with h5py.File(trig_file, "r") as f:` context just in case anything goes wrong? Or even just grab the relevant `triggers`, `times`, and `snrs` arrays under the context and then do all the manipulation after they're loaded and the file is closed?